code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -138,15 +138,14 @@
 /obj/item/clothing/head/helmet/space/syndicate/contract
 	name = "contractor helmet"
 	desc = "A specialised black and gold helmet that's more compact than its standard Syndicate counterpart. Can be ultra-compressed into even the tightest of spaces."
-	slowdown = 0.55
 	w_class = WEIGHT_CLASS_SMALL
 	icon_state = "syndicate-contract-helm"
 	item_state = "syndicate-contract-helm"
 
 /obj/item/clothing/suit/space/syndicate/contract
 	name = "contractor space suit"
 	desc = "A specialised black and gold space suit that's quicker, and more compact than its standard Syndicate counterpart. Can be ultra-compressed into even the tightest of spaces."
-	slowdown = 0.55
+	slowdown = 1

```

Comment:
did you intentionally drop the 0.1

Reply:
yes, it's dumb, i can give it 1.1 if you want but like why

Reply:
you didn't doc the change so I figured I would ask :smirk: